liquidator/abi/OneSplit.json

```diff
@@ -0,0 +1,238 @@
+[
```

Comment:
Could we write a minimal solidity interface for this instead? Something like:
```
pragma solidity ^0.6.0;

import "@openzeppelin/contracts/token/ERC20/IERC20.sol";


abstract contract OneSplitInterface {
    function getExpectedReturn(
        IERC20 fromToken,
        IERC20 destToken,
        uint256 amount,
        uint256 parts,
        uint256 flags // See constants in IOneSplit.sol
    ) public virtual view returns (uint256 returnAmount, uint256[] memory distribution);

    function swap(
        IERC20 fromToken,
        IERC20 destToken,
        uint256 amount,
        uint256 minReturn,
        uint256[] memory distribution,
        uint256 flags
    ) public virtual payable returns (uint256 returnAmount);
}
```

This would make it easier to read/understand the types involved (abis are not human-readable) and it only includes the functions we use, so the other clutter can be left out. It would also make it pretty trivial to build a solidity mock contract for testing.

These functions were copied with very slight modifications (for 0.6) from [here](https://github.com/CryptoManiacsZone/1inchProtocol/blob/master/contracts/IOneSplit.sol).

You could put this under `core/contracts/common/interfaces` for now and import it like you would any other abi in our system.